app/views/questions/triage_income_level/edit.html.erb

```diff
@@ -50,7 +50,7 @@
       ) %>
 
     <div class="spacing-above-60">
-      <%= f.submit t("general.continue"), class: "button button--primary button--wide text--centered" %>
+      <%= f.continue %>
```

Comment:
FYI to reviewers: `.button` makes the text centered, so `text--centered` is redundant.